modules/ocf_mirrors/files/collect-mirrors-stats

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python3
+import argparse
+import os
+import traceback
+from datetime import date
+from datetime import datetime
+from datetime import timedelta
+from pathlib import Path
+
+from ocflib.lab.stats import get_connection
+from ocflib.lab.stats import humanize_bytes
+
+OCFSTATS_PWD = os.environ.get('OCFSTATS_PWD', None)
+MIRRORS_DATA_PATH = Path('/opt/mirrors/ftp')
+APACHE_LOG_PATH = Path('/var/log/apache2')
+APACHE_LOG_FILES = [
+    Path('mirrors.ocf.berkeley.edu_access.log'),
+    Path('mirrors.ocf.berkeley.edu-ssl_access_ssl.log')
+]
+RSYNC_LOG_PATH = Path('/var/log/rsync')
+
+
+def log_to_mysql(projects, dt=None, quiet=False):
+    log_date = dt or date.today()
```

Comment:
Nit: maybe just `dt=date.today()`?

Reply:
Or just make it a required param, since it's always provided as an argument in this script?